src/app/pages/user-profile/password-modal/password-modal.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, inject } from '@angular/core';
+import { NonNullableFormBuilder, ReactiveFormsModule, Validators } from '@angular/forms';
+import { MatDialogModule, MatDialogRef } from '@angular/material/dialog';
+import { passwordValidation } from '../../../components/form-error-message/password-validation';
```

Comment:
```suggestion
import { passwordValidation } from '../../../validators/password-validation';
```
passwordValidation  were moved